src/main/resources/application.yaml

```diff
@@ -5,6 +5,9 @@ spring:
       ddl-auto: validate
     open-in-view: false
   datasource:
+    url: jdbc:postgresql://localhost:5432/discounts
```

Comment:
U do not need to change this part:
  datasource:
    url: jdbc:postgresql://localhost:5432/
is enough

Reply:
done